app/vendor/vendor.js

```diff
@@ -10,7 +10,7 @@ var angularSanitize = require('angular-sanitize');
 var angularBreadcrumb = require('angular-breadcrumb');
 var bootstrap = require('bootstrap');
 var domReady = require('domready');
-var AgGrid = require('ag-grid');
+var AgGrid = require('../../node_modules/ag-grid/dist/ag-grid.noStyle');
```

Comment:
Probably. But I'm actually thinking of removing ag-grid entirely. It seems like we don't do a whole lot with it and it adds several hundred kb of js. Either way, I'll do something about this in the next pr.